features/step_definitions/filtering_steps.rb

```diff
@@ -51,6 +51,12 @@
   expect(page).to have_css('.filtered-results .document', count: 1)
 end
 
+
+# And("I see email and feed sign up links") do
```

Comment:
This test is commented out because I couldn't get it to work.

Since the new code works in a way that (I think at least) is very fragile, by looking for a[href*=".atom" or "email-signup"] I kind of wanted some sort of functional test to confirm that its working as intended. 
I'm not really sure how to achieve this within cucumber though, both to confirm that its not finding the wrong thing, and that it is finding the right thing, though the chance of having more than one anchor with a href containing .atom or email-signup is probably pretty low.

Reply:
It sounds worthwhile so it would be cool to get this working.

I think this may do the trick:

```
# in features/finders.feature:
  Scenario: Email links
    When I view the news and communications finder
    Then I see email and feed sign up links
    And I select a Person
    And I filter the results
    Then I see email and feed sign up links with filters applied


# in features/step_definitions/filtering_steps.rb:
And("I see email and feed sign up links") do
  expect(page).to have_link('Get email alerts', href: "/news-and-communications/email-signup?")
  expect(page).to have_link('Subscribe to feed', href: "/news-and-communications.atom")
end

And("I see email and feed sign up links with filters applied") do
  expect(page).to have_link('Get email alerts', href: "/news-and-communications/email-signup?people%5B%5D=rufus-scrimgeour")
  expect(page).to have_link('Subscribe to feed', href: "/news-and-communications.atom?order=updated-newest&amp;people%5B%5D=rufus-scrimgeour&amp;public_timestamp%5Bfrom%5D=&amp;public_timestamp%5Bto%5D=")
end
```